File: model/src/main/scala/net/server/channel/packet/UseSuperMegaphonePacket.scala
package net.server.channel.packet

class UseSuperMegaphonePacket(private var _position: Short, private var _itemId: Int, private var _message:String, private var _ear:Boolean) extends AbstractUseCashItemPacket(_position, _itemId) {
  def message: String = _message

  def ear: Boolean = _ear
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/use/UseSuperMegaphonePacket.java
package net.server.channel.packet.cash.use;

public class UseSuperMegaphonePacket extends AbstractUseCashItemPacket {
   private final String message;

   private final Boolean ear;

   public UseSuperMegaphonePacket(Short position, Integer itemId, String message, Boolean ear) {
      super(position, itemId);
      this.message = message;
      this.ear = ear;
   }

   public String message() {
      return message;
   }

   public Boolean ear() {
      return ear;
   }
}
